packages/xo-server/src/_handleBackupLog.mjs

```diff
@@ -1,3 +1,19 @@
+import moment from 'moment-timezone'
+import humanFormat from 'human-format'
+
+let vmBackupInfo = {}
```

Comment:
Avoid using a global variable, create a new map each time you run a backup job and pass it to `handleBackupLog`, this will prevent unnecessary data from staying there and memory leaks.